server/gql/models/tests/books/query.test.js

```diff
@@ -0,0 +1,311 @@
+import get from 'lodash/get';
+import { getResponse, mockDBClient, resetAndMockDB } from '@utils/testUtils';
+import { authorsTable, languagesTable, publishersTable, booksTable } from '@utils/testUtils/mockData';
+
+describe('Language graphQL-server-DB query tests', () => {
+  const bookId = parseInt(booksTable[0].id);
+  const publisher = publishersTable[0].name;
+  const language = languagesTable[0].language;
+  const genre = booksTable[0].genres;
+
+  const bookOneFromId = `
```

Comment:
lets move the queries to another file. The lint rules should've prevented this however we have a rule that says the lines shouldn't be more than 300 per file
https://github.com/sagar-wednesday/node-express-graphql-template/blob/develop/.eslintrc.js#L39